npm/private/npm_package_store.bzl

```diff
@@ -244,24 +244,29 @@ def _npm_package_store_impl(ctx):
                         tar_exclude_package_contents.append("--exclude")
```

Comment:
if you're using `args` now you can just `args.add_all` and use before_each + map_each param to do the filtering, don't need to expand this here